_layouts/base.html

```diff
@@ -132,11 +126,10 @@ <h1><a href="{{ site.baseurl }}/">{{ site.title }}</a></h1>
 
       function loadSearch(lang, site) {
         docsearch(Object.assign(
-          { algoliaOptions: { facetFilters: ['site: ' + site] } },
+          { searchParameters: { facetFilters: ['site: ' + site] } },
+          { placeholder: "{% if t.pagecontent.search %}{{ t.pagecontent.search }}{% else %}Search{% endif %} {% if page.search_name %}{{ page.search_name }}{% else %}Homebrew{% endif %}" },
           {{ layout.algolia | jsonify }}
         ));
-        window.location.hash || document.querySelector('#search-bar').focus();
-        document.querySelector('#search-bar').value = new URLSearchParams(window.location.search).get('search');
```

Comment:
This was added for Homebrew/formulae.brew.sh#234 it's worth noting

Reply:
Let's try it.